JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/converter/DescriptionLogicGenerator.java

```diff
@@ -143,26 +143,27 @@ public static void initNamespaces() {
     public static String convertToLatex(String dlSyntax) {
         initNamespaces();
         String latexFormat = "% add \\usepackage{seqsplit} to your preamble \n\\seqsplit{" + dlSyntax
-        .replace("⊓", "$\\sqcap$")
-        .replace("⊔", "$\\sqcup$")
-        .replace("¬", "$\\neg$")
-        .replace("≡", "$\\equiv$")
-        .replace(">", "$>$")
-        .replace("<", "$<$")
-        .replace("=", "$=$")
-        .replace("⊑", "$\\sqsubseteq$")
-        .replace("⊒", "$\\sqsupseteq$")
-        .replace("⊤", "$\\top$")
-        .replace("⊥", "$\\bot$")
-        .replace("∀", "$\\forall$")
-        .replace("∃", "$\\exists$")
-        .replace("→", "$\\rightarrow$")
-        .replace("⇒", "$\\Rightarrow$")
-        .replace("⇔", "$\\Leftrightarrow$")
+        .replace("⊓", "{$\\sqcap$")
```

Comment:
There is a "}" missing?

Reply:
Addressed in https://github.com/cambridge-cares/TheWorldAvatar/pull/736/commits/c5469504e09adfdfefdfc2f36a88dafa0478f253